src/core/env.js

```diff
@@ -8,18 +8,23 @@
 
 var env = {};
 
+var miniAppEnv = {
+    browser: {},
+    os: {},
+    node: false,
+    canvasSupported: true,
+    svgSupported: false,
+    touchEventsSupported: true,
+    domSupported: false
```

Comment:
It would be better if you add `wxa: false, swan: false` here for default values.

Reply:
Reference the detect of browser . `env.firefox` has not default values . so do `env.wxa`.